HostSupport/src/ofxhImageEffect.cpp

```diff
@@ -341,6 +342,7 @@ namespace OFX {
         { kOfxImageEffectPropSupportsTiles,     Property::eInt,        1, false, "1" }, // OFX 1.4
 #ifdef OFX_SUPPORTS_OPENGLRENDER
         { kOfxImageEffectPropOpenGLRenderSupported, Property::eString, 1, false, "false"}, // OFX 1.4
+        { kOfxOpenGLPropPixelDepth, Property::eString,  0, false, "" },
```

Comment:
this should only be in the descriptor, not in the instance (just like kOfxImageEffectPropSupportedPixelDepths)